src/FrameHelper.cpp

```diff
@@ -247,7 +264,9 @@ namespace frame_helper
         dst.init(src,false);
         const cv::Mat cv_src = FrameHelper::convertToCvMat(src);
         cv::Mat cv_dst = FrameHelper::convertToCvMat(dst);
-        remap(cv_src, cv_dst, calibration.map1, calibration.map2, cv::INTER_CUBIC);
+        cv::InterpolationFlags interpolation_flag;
```

Comment:
* Inconsistent naming. Keep equivalent names for equivalent things
* You forgot to call `undistortAlgorithmToCV` doing your refactoring

```
auto undistort_algorithm_cv = undistortAlgorithmToCV(undistort_algorithm);
```